ui-common/src/components/UI/StatusCode.tsx

```diff
@@ -37,9 +37,9 @@ export function getClassification(statusCode: number): string {
 
     // 1 - 16 HTTP/2 (gRPC) status codes
```

Comment:
this comment is probably not relevant is we are not changing the status to be 1-16 (from `Grpc-Status` header value)